.classpath

```diff
@@ -52,34 +50,31 @@
 	<classpathentry kind="lib" path="extensions/exquery/restxq/lib/exquery-restxq-1.0-SNAPSHOT.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="lib" path="lib/core/antlr-2.7.7.jar"/>
-	<classpathentry kind="lib" path="lib/core/excalibur-cli-1.0.jar"/>
 	<classpathentry kind="lib" path="lib/core/jline-0.9.94.jar"/>
 	<classpathentry kind="lib" path="lib/core/log4j-api-2.8.2.jar"/>
 	<classpathentry kind="lib" path="lib/core/log4j-core-2.8.2.jar"/>
 	<classpathentry kind="lib" path="lib/core/log4j-slf4j-impl-2.8.2.jar"/>
 	<classpathentry kind="lib" path="lib/core/log4j-jul-2.8.2.jar"/>
+	<classpathentry kind="lib" path="lib/core/jargo-0.4.2-SNAPSHOT-7fa1d04.jar" sourcepath="lib/core/jargo-src.zip"/>
```

Comment:
for me, lib/core/jargo-src.zip does not exist?

Reply:
Yes, remove it. 
I used it to debug then removed it, and forgot to correct classpath.